lib/Word_Lib/Word_Lemmas.thy

```diff
@@ -496,7 +496,7 @@ next
       ultimately show ?thesis
         using \<open>y < LENGTH('a)\<close>
         by (auto simp add: drop_bit_eq_div word_less_nat_alt unat_div unat_word_ariths
-                           shiftr_def shiftl_def)
+          shiftr_def shiftl_def)
```

Comment:
Indentation

Reply:
Hm, these are noise from the AFP. I guess we should fix them, otherwise we'll accumulate them.

Reply:
Done. Thanks for spotting these.